Add years to external article links (#39701)
mdn/content: files/en-us/web/css/font-smooth/index.md

## See also

> - [Please Stop "Fixing" Font Smoothing](https://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/) via UsabilityPost (2012)

Reviewer: ```suggestion
- [Please Stop "Fixing" Font Smoothing](https://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/) by Dmitry Fadeyev (2012)
```

> - [Laissez-faire font smoothing and anti-aliasing](https://www.zachleat.com/web/font-smooth/) via Zachleat (2017)

Reviewer: ```suggestion
- [Laissez-faire font smoothing and anti-aliasing](https://www.zachleat.com/web/font-smooth/) by Zach Leatherman (2017)
```
